### ISSUE TITLE
 on ios, when i put my iphone in airplane mode it doesn't work

### ISSUE THREAD

**Issue by thanhnvpk01168** [birdofpreyru/react-native-static-server#7]
on android everything works fine even in airplane mode, but not on ios.
Can you help me see this problem?
thanks in advance

**Comment by birdofpreyru** [comment]
This sounds similar #2 by @vyrotek. I briefly checked it in my project, and saw no issue. However, probably a deeper testing is needed, and if there is a problem, it is probably in the underlying ios server library, on which this RN lib relies.

Hey @vyrotek, did you look more into it? Do you still have the issue, or found it was some other problem in your case.

**Comment by vyrotek** [comment]
We ultimately addressed our server issue with just the "localOnly:true" setting and made sure we used the full URL return from the Server Start method. Other issues were our own problems.

Although, while researching this we did stumble onto various comments and info about the server URLs. It seems that localOnly:true returns and expects localhost:port URLs but localOnly:false returns you an ip-address:port. I suspect there is something deeper happening with the server sometimes on iOS when localOnly is false and switching to airplane mode. I saw some people experiment with replacing "localhost" with "127.0.0.1" a long time ago.

Something else to watch out for are iOS settings being set to continue to use WIFI even in Air Plane Mode or the device trying to use the Mac-connected cable for a network connectivity. We kept getting inconsistent test results because of various device settings.
